pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -0,0 +1,33 @@
+[id="creating-a-product_{context}"]
+= Creating a product
+
+A product must exist in {ProductName} before you can assign a product name to the metadata of a module or assembly. If the product doesn't already exist, you need to create one.
+
+.Prerequisites
+
+* You are logged in.
+* You have Publisher permissions.
+
+.Procedure
+
+. Navigate to menu:Products[New Product].
+
+. Enter the following information:
+  Product Name:: This is the text that appears in {ProductShortName}, the document preview, and in user-facing content.
+  Product UrlFragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+  Product Description (optional):: A description of the product.
+
+. Click btn:[Save].
+
+After creating a product, you need to create a product version.
+
+.Example
+
+Product Name:: .NET Core
+Product UrlFragment:: `net_core` 
+Product Description:: Documentation for the .NET Core framework.
+
+Entering the above information creates a {ProductShortName} product named `.Net Core`. The URL for documentation that is associated with this product includes `net_core` to identify the product. For example, after creating the product version `4.4-beta` and assigning a module the .NET Core product and version 4.4-beta, the resulting URL for that module is: `https://access.stage.redhat.com/documentation/en-us/topic/net_core/4.4-beta/_module-UUID_`
```

Comment:
The product would be named `.NET Core`. The last sentence seems slightly malformed, could we reword it some?

```adoc
Entering the above information creates a {ProductShortName} product named `.NET Core`. The URL for documentation associated with this product includes the `net_core` URL fragment to identify the product.

For example, after creating a product version of `4.4-beta` and assigning a module to the .NET Core product with a version of `4.4-beta`, the resulting URL for that module is:
```

Also, are we using attributes for the URLs in the documentation? If so, I'd use that instead of hardcoding the URL for the described module.

Reply:
@robin-owen There aren't any attributes for this right now, but I agree.